src/CardDeck.php

```diff
@@ -147,7 +152,7 @@ public function setModel(Model $model, array $fields = null, array $extra = null
         if ($count) {
             foreach ($this->model as $m) {
                 /** @var Card */
-                $c = $this->cardHolder->add(Factory::factory([$this->card], ['useLabel' => $this->useLabel, 'useTable' => $this->useTable]));
+                $c = $this->cardHolder->add(Factory::factory((array) $this->card, ['useLabel' => $this->useLabel, 'useTable' => $this->useTable]));
```

Comment:
never use `(array)` cast, change the def. value to be an array seed instead

Reply:
Ok, will work on that a bit more probably tomorrow. Looks like there are more seed related things to fix in this (and Card) classes for consistency with other views and also to ease extending.